src/Contexts/TodoContext.tsx

```diff
@@ -32,8 +37,31 @@ export const TodoProvider: React.FC<TodoProviderProps> = ({ children }) => {
   const [tasks, setTasks] = useState<Task[]>([]);

```

Comment:
Atualizar o valor inicial do seu tasks

Basicamente, vamos procurar no localStorage. 

- Se existir vamos popular
- Se nao, vamos passar []

```suggestion
  const [tasks, setTasks] = useState<Task[]>(() => {
    const storageTasks = localStorage.getItem("@TodoApp:tasks");
    if (storageTasks) {
      return JSON.parse(storageTasks);
    }
    return [];
  });
```